tests/conftest.py

```diff
@@ -24,7 +26,7 @@ def pytest_runtest_makereport(item, call):
 
 @pytest.fixture
 def slack_channel() -> SlackChannel:
-    if not "PYTEST_SLACK_TOKEN" in os.environ or not "PYTEST_SLACK_CHANNEL" in os.environ:
+    if "PYTEST_SLACK_TOKEN" not in os.environ or "PYTEST_SLACK_CHANNEL" not in os.environ:
```

Comment:
verify pytests for slack pass - its part of git automated tests

Reply:
It is the same - in python (because of readability) it is preferred to use `if "Something" not in collection`, not `if not "Something" in collection` - but result is the same. This one is also a warning from flake8.